providers/google/docs/changelog.rst

```diff
@@ -27,6 +27,24 @@
 Changelog
 ---------
 
+17.0.0
```

Comment:
Please merge the changes to 16.0
We didn't release 16.0 yet

Reply:
Updated